test/html_test.rb

```diff
@@ -44,6 +44,6 @@ def content
     # combine all white space
     html_str = html_str.gsub(/\s+/, ' ')
     # clear any white space between block level tags and other content
-    html_str.gsub(%r{\s*<(/?(?:h\d|div|p|br|ul|ol|li).*?)>\s*}, '<\1>')
+    html_str.gsub(%r{\s*<(/?(?:h\d|div|p|br|ul|ol|li|a).*?)>\s*}, '<\1>')
```

Comment:
The anchor tag isn't block level, so we want to preserve it's surrounding whitespace otherwise well get stuff like "textwww.google.comtext" instead of the desired "text  www.google.com  text".

Also, we will need to add some links to the `html_test_content` fixture once the testing issue with SecureRandom is squared up. Preferably one without any styles and one with some basic formatting. This integration test is probably our most important test since it allows us to see how MS Word will actually interpret things. Especially if/when we find a case where Microsoft deviated from the formal OOXML spec.